lib/index.js

```diff
@@ -58,13 +60,28 @@ function AppEngine() {
         stringify: this_.loggingStringify_,
         // TODO(rrch): Enable log rotation once the fluentd config has been
         // updated to accept the kind of filenames winston creates.
-        // maxFiles: 3,
-        maxsize: 100 * 1024 * 1024
+        maxFiles: this_.options_.loggerMaxFiles,
+        maxsize: this_.options_.loggerMaxFileSize
       })
     ]
   });
+  this.httpAgent_ = new http.Agent({
+    maxSockets: this.options_.httpAgentMaxSockets
+  });
+  if (this.options_.exportPublicFunctions) {
+    this.exportAll_();
+  }
 }
 
+AppEngine.prototype.defaultOptions_ = {
+  logSizeThreshold: 1024 * 1024, // 1 MB
+  logMaxTimestampDeltaMillis: 60000, // 60 seconds
+  loggerMaxFiles: 1,
+  loggerMaxFileSize: 100 * 1024 * 1024, // 100 MB
+  httpAgentMaxSockets: 100,
+  exportPublicFunctions: true
```

Comment:
We should add a comment it's building the surface layer. I'd go with different name such as initialize or build or whatever as we discussed earlier.